### ISSUE TITLE
Task incomplete for RedHat >=7

### ISSUE THREAD

**Issue by ptobler** [jtyr/ansible-grub_cmdline#1]
Dear Jiri,

I just tried out your grub_cmdline role on Centos 7. Instead of adding the requested arg, it replaced all existing args by the one to add. I had a quick lock at main.yml and it seems that there is no task getting the current content of GRUB_CMDLINE_LINUX.

Cheers, Philipp

**Comment by jtyr** [comment]
That's right. This role doesn't allow to "add" args for CentOS 7. It overwrites all args by the newly defined args. If you see a feasible solution how to read the already defined args and merge them with the new args before writing them into the file, please suggest it here or even better create a PR.

**Comment by ptobler** [comment]
Thanks for your quick reply! I think i'll add the needed args during kickstart installation instead, that's good enough for me.
